python/ray/autoscaler/aws/node_provider.py

```diff
@@ -218,14 +228,20 @@ def terminate_nodes(self, node_ids):
             self.tag_cache.pop(node_id, None)
             self.tag_cache_pending.pop(node_id, None)
 
-    def _node(self, node_id):
-        if node_id not in self.cached_nodes:
-            self.nodes({})  # Side effect: should cache it.
+    def _get_node(self, node_id):
+        self.nodes({})  # Side effect: fetches and caches the node.
 
         assert node_id in self.cached_nodes, "Invalid instance id {}".format(
             node_id)
+
         return self.cached_nodes[node_id]
 
+    def _get_cached_node(self, node_id):
```

Comment:
```suggestion
    def _get_cached_node(self, node_id):
         """Return info from this node from cache if possible, otherwise fetching it."""
```